scripts/drone/steps/lib.star

```diff
@@ -1021,6 +1021,57 @@ def integration_benchmarks_step(name, environment = None):
 
     return integration_tests_steps("{}-benchmark".format(name), cmds, environment = environment)
 
+def e2e_apiserver_tests_step():
+    cmds = [
+        'apk add --update --no-cache build-base make git coreutils bash docker gcc libc-dev',
+        'export GRAFANA_IMAGE=grafana/grafana-enterprise-dev:$(cat $OSS_BUILD_VERSION_FILE)',
```

Comment:
is `grafana-enterprise-dev` correct for OSS?

Reply:
this was a typo, good catch